### PR TITLE
Fluent calendar

### PR DESCRIPTION
**Pull request by TeyaVes**
related to: https://github.com/telerik/kendo-themes-private/issues/2

### CODE REVIEW

File: packages/fluent/scss/core/_variables.scss [hunk 1]
```diff
@@ -60,14 +71,34 @@ $kendo-letter-spacing: normal !default;
 
 $default-scrollbar-width: 17px !default;
 
+/// Border radius for all components.
+$kendo-border-radius: map-get( $kendo-spacing, thin ) !default; // map-get( $kendo-spacing, 1 )
+$kendo-border-radius-sm: calc( #{$kendo-border-radius} / 2 ) !default;
+$kendo-border-radius-md: calc( #{$kendo-border-radius} ) !default;
```
Comment (TeyaVes) [inline packages/fluent/scss/core/_variables.scss:77]: remove calc

File: packages/fluent/scss/calendar/_theme.scss [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+        // Today navigation
+        .k-nav-today {
+            color: var(--kendo-calendar-today-nav-text, #{$kendo-calendar-today-nav-text});
+
+            &:hover,
+            &.k-hover,
+            &:focus,
+            &.k-focus {
+                color: var(--kendo-calendar-today-nav-hover-text, #{$kendo-calendar-today-nav-hover-text}) $kendo-calendar-today-nav-hover-text;
```
Comment (epetrow) [inline packages/fluent/scss/calendar/_theme.scss:32]: Do we need the second $kendo-calendar-today-nav-hover-text variable here?

File: packages/fluent/scss/calendar/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+/// Vertical padding of the infinite calendar.
+/// @group calendar
+$kendo-infinite-calendar-view-padding-y: 0px !default;
+$kendo-infinite-calendar-view-height: calc( $kendo-calendar-cell-size * 9 ) !default;
+
+
+// Range calendar
+/// Background color of the range selection in the calendar.
+/// @group calendar
+$kendo-calendar-range-bg: get-theme-color-var(neutral-30) !default;
```
Comment (epetrow) [inline packages/fluent/scss/calendar/_variables.scss:257]: Add spacing here for consistency

File: packages/fluent/scss/core/_variables.scss [hunk 4]
```diff
@@ -60,14 +71,34 @@ $kendo-letter-spacing: normal !default;
 
 $default-scrollbar-width: 17px !default;
 
+/// Border radius for all components.
+$kendo-border-radius: map-get( $kendo-spacing, thin ) !default; // map-get( $kendo-spacing, 1 )
+$kendo-border-radius-sm: calc( #{$kendo-border-radius} / 2 ) !default;
+$kendo-border-radius-md: #{$kendo-border-radius} !default;
```
Comment (epetrow) [inline packages/fluent/scss/core/_variables.scss:77]: Do we need interpolation here?

File: packages/fluent/scss/core/color-system/_theme.scss [hunk 5]
```diff
@@ -32,8 +47,15 @@ $kendo-box-shadow-depth-3: 0px 1.2px 3.6px rgba( $black, 0.1 ), 0px 6.4px 14.4px
...
 
 // Link
-$kendo-link-text: get-theme-color-var(primary-130) !default;
-$kendo-link-hover-text: get-theme-color-var(primary-160) !default;
+$kendo-link-text: get-theme-color-var( primary-130 ) !default;
+$kendo-link-hover-text: get-theme-color-var( primary-160 ) !default;
+
+// Validator
+$kendo-invalid-bg: null !default;
+$kendo-invalid-text: get-theme-color-var(error-190) !default;
```
Comment (epetrow) [inline packages/fluent/scss/core/color-system/_theme.scss:55]: Add spacing here and below

**Review by Juveniel — APPROVED** [review]

**Pull request merged**